### PR TITLE
Typo fixes and code cleanup

### PR DESCRIPTION
**Pull request by tvolk131**
## Change Description
Various documentation typo fixes, a simple code modification, and removal of two empty files

## Steps to Test
None

### CODE REVIEW

**Review by yyforyongyu — CHANGES_REQUESTED** [review]
Thank you for the PR! Please refer to [our contribution guide](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#ideal-git-commit-structure) regarding how to structure your commits and commit messages. And I'd suggest separating the changes into two commits if you are touching both the docs and the code.

File: channeldb/payment_control.go [hunk 1]
```diff
@@ -429,8 +429,6 @@ func (p *PaymentControl) updateHtlcKey(paymentHash lntypes.Hash,
 
 	var payment *MPPayment
 	err := kvdb.Batch(p.db.Backend, func(tx kvdb.RwTx) error {
-		payment = nil
```
[the remark this answers is not in the document]
Comment (tvolk131) [reply channeldb/payment_control.go]: Ahh gotcha. Done.

**Review by guggero — APPROVED** [review]
LGTM, thanks!

**Review by yyforyongyu — APPROVED** [review]
LGTM🥳

**Pull request merged**